deploy-tool/config/config.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"github.com/pelletier/go-toml"
+)
+
+type DeployConfig struct {
+	DNS     DNSConfig
+	ACME    ACMEConfig
+	Storage StorageConfig
+}
+
+type DNSConfig struct {
+	Provider         string
+	CloudflareConfig *CloudflareConfig `toml:"cloudflare"`
+}
+
+type CloudflareConfig struct {
+	APIKey string `toml:"api_key"`
+}
+
+type ACMEConfig struct {
+	Email               string
+	ACMEApiEndpoint     string `toml:"acme_api_endpoint" default:"https://acme-v02.api.letsencrypt.org/directory"`
+	SubscriberAgreement bool   `toml:"subscriber_agreement"`
+}
+
+type StorageConfig struct {
+	Driver     string `default:"kubernetes"`
+	Filesystem *FilesystemConfig
+	Kubernetes *KubernetesConfig
+}
+
+type FilesystemConfig struct {
+	Path string `default:"./deploy_tool_output"`
+}
+
+type KubernetesConfig struct {
+	Namespace string `default:"prio-server"`
```

Comment:
Putting the ACME credential in a Kubernetes namespace seems like a fine idea, but if the idea is that this will eventually be used by a key rotation cronjob running in that same namespace, maybe "key-rotator" would be more appropriate.